Dockerfile

```diff
@@ -65,10 +64,13 @@ RUN cp /opt/TangoService/Tango/deployment/config/nginx.conf /etc/nginx/nginx.con
 RUN cp /opt/TangoService/Tango/deployment/config/supervisord.conf /etc/supervisor/supervisord.conf
 RUN cp /opt/TangoService/Tango/deployment/config/redis.conf /etc/redis.conf
 
+# Add user
+RUN useradd -ms /bin/bash autolab
+
 # Reload new config scripts
 CMD ["/usr/bin/supervisord", "-c", "/etc/supervisor/supervisord.conf"]
 
-
+CMD ["/usr/bin/supervisord"]
```

Comment:
The above command on line 71 should start the supervisor with the specified config file. What is the purpose of executing supervisor without config?

Reply:
this is was a sloppy mistake it doesn't need to be there nor should it but it does not affect the container overall as far as i can tell i will remove it.